src/stories/baseComponents/color.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Divider, Typography } from '@material-ui/core';
+
+import { colorPalette, defaultStyles, StyleTyping } from './utils';
+
+const styles: StyleTyping = {
+  ...defaultStyles,
+  example: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    margin: '1rem 0',
+    padding: '0 0.5rem',
+  },
+};
+
+export default function Colors() {
+  return (
+    <React.Fragment>
+      {Object.keys(colorPalette).map((color, index) => {
+        return (
+          <React.Fragment>
+            <div style={styles.example} key={index}>
+              <Typography variant="body1">Unknown</Typography>
+              <div>
+                <Typography variant="body1">{`Color code: ${colorPalette[color].backgroundColor}`}</Typography>
```

Comment:
Just to confirm @koonpeng , the hardcoded color that you were referring to is on this line?